rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -194,6 +225,56 @@ impl LodeSphericalExpansion {
             }
         }
     }
+
+    fn compute_density_fourrier(&self, k_vectors: &[KVector]) -> Array1<f64> {
+        let mut fourrier = Vec::new();
+        fourrier.reserve(k_vectors.len());
+
+        let potential_exponent = self.parameters.potential_exponent;
+        let smearing_squared = self.parameters.atomic_gaussian_width * self.parameters.atomic_gaussian_width;
+
+        if potential_exponent == 0 {
+            let factor = (4.0 * std::f64::consts::PI * smearing_squared).powf(0.75);
+
+            for k_vector in k_vectors {
+                let value = f64::exp(-0.5 * k_vector.norm * k_vector.norm * smearing_squared);
+                fourrier.push(factor * value);
+            }
+
+            // For Gaussian potentials, the Fourier transform at k = 0 is finite and contributes to the coefficients
+            // self.radial_integral.compute(0.0);
+            // let mut value = 1.0 ; // put k=0 value here
```

Comment:
Sorry for asking again, but how can I access the radial integral value for k=0. 🫣

Reply:
it is not computed currently, but you should be able to do something like `self.radial_integral.compute(0.0)` and then access `self.radial_integral.values[[l, n]]`. If it is not clear we can have a quick chat tomorrow!

Reply:
Okay, yes we can have a quick chat tomorrow.